BibTeX.js

```diff
@@ -292,6 +294,12 @@ function processField(item, field, value, rawValue) {
 		} else {
 			item.publicationTitle = value;
 		}
+	} else if(field == "journaltitle") {
+		if(item.publicationTitle) {
+			item.journalAbbreviation = value;
+		} else {
+			item.publicationTitle = value;
+		}
```

Comment:
I don't think we need this logic here. The logic make sense for bibtex, where it is possible to have `journal` and `fjournal` and any of them are processed first. However, in BibLaTeX there is `shortjournal` and `journaltitle` for that and `journaltitle` is always mapped to `publicationTitle`. Thus, you can add it to the `inputFieldMap` in line https://github.com/twsh/translators/blob/9de98ca98b5548c53987ed839f20bd20600ab61f/BibTeX.js#L167 maybe with a comment that indicates that the following fields are for importing BibLaTeX.